opentelekomcloud/services/iam/resource_opentelekomcloud_identity_user_v3.go

```diff
@@ -209,7 +211,7 @@ func resourceIdentityUserV3Update(ctx context.Context, d *schema.ResourceData, m
 		}
 	}
 
-	if d.HasChange("email") && d.Get("send_welcome_email").(bool) {
+	if updateOpts.Email != "" && d.Get("send_welcome_email").(bool) {
```

Comment:
Same in create pls

Reply:
this check is performed with RequiredWith field for "send_welcome_email"